src/Features/Core/Portable/PullMemberUp/MembersPuller.cs

```diff
@@ -378,7 +381,7 @@ private static ISymbol GetSymbolsToPullUp(MemberAnalysisResult analysisResult)
             destinationEditor.ReplaceNode(destinationEditor.OriginalRoot, (node, generator) => addImportsService.AddImports(
                 destinationEditor.SemanticModel.Compilation,
                 node,
-                node.GetCurrentNode(newDestination),
+                node.GetAnnotatedNodes(s_destinationNodeAnnotation).FirstOrDefault(),
```

Comment:
this was the bad line.  They were basically trying to find this totally fresh node in the current root.  but that totally fresh node was already morphed into a different node when teh replacement happened.  Trivial fix is to just use our own annotation to track it, instead of the editor tracking *all* nodes.